src/Canvanoid.js

```diff
@@ -17,39 +17,93 @@ export default class Canvanoid {
 					  delta: null }
 
 	    this.messagePanel = null;
-		this.score = null;
+		this.instructionsPanel = null;
+		this.scorePanel = null;
 
 		this.state = null; 
 		this.board = null;
 		this.balls = null;
 		this.vaus = null; 
+		this.pause = null;
 	}
 	
 	start() {	
+		// Game Elements
 		this.state = new State();
-		
 		this.board = new Board();
 		this.board.setStage(this.state.stage);
 
-		this.balls = [ new Ball(this.board.x + this.board.w/2, this.board.y + this.board.h/2 + 100) ];
-		this.vaus = new Vaus(this.board.x + this.board.w/2 - 50, 
-              				 this.board.y + this.board.h - 50);
-		this.vaus.start();
+		this.balls = [ new Ball(this.board.position.x + this.board.width/2, this.board.position.y + this.board.height/2 + 100) ];
+		this.vaus = new Paddle(this.board.position.x + this.board.width/2 - 50, 
+              				   this.board.position.y + this.board.height - 50);
+		this.interruptions();
 
-		this.score = new Score(this.board.x + this.board.w - 100, 
-		                       this.board.y + this.board.h + 30);
+		// Interface Elements
+		this.messagePanel = new Panel(this.board.position.x + this.board.width/2, 
+							          this.board.position.y + this.board.height/2,
+							   		  this.ctx);
 		
-		this.messagePanel = new Panel( this.board.x + this.board.w/2 - 150, 
-							           this.board.y + this.board.h/2 - 100);
+		this.instructionsPanel = new Panel(this.messagePanel.initialPosition.x, this.messagePanel.initialPosition.y + 50,
```

Comment:
Same here I would encapsulate this in a function (and maybe all this I would isolate it in a different file (module) or class).


There is a principle called: Single level of abstraction level, you should read this code like a book, and it should have the same level of abstraction, not going deeper into coordinates and offsets

http://principles-wiki.net/principles:single_level_of_abstraction